stable/rancher/templates/certificate.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if eq .Values.ingress.tls "letsEncrypt" -}}
```

Comment:
Why not take advantage of cert manager's ingress-shim like most other charts do?

Reply:
I found the cert-manager docs a little vague.  I thought the shim was the temporary endpoint that gets set up to return the challenge for LE and part of the cert-manager chart?  

If there is a better way, I'm all for it.  Can you point me at an example?